src/kafka-topics/list/topics-list.html

```diff
@@ -39,8 +40,8 @@
     </div>
 
     <!--Pagination-->
-    <div flex>
-      <dir-pagination-controls pagination-id="topics" style="margin:0;"></dir-pagination-controls>
+    <div flex class="leftlistPagination">
+      <dir-pagination-controls on-page-change="pageChanged(newPageNumber)" pagination-id="topics" style="margin:0;"></dir-pagination-controls>
```

Comment:
is this different from `getTopicPage` in the `control-topics-list.html` pagination? better have a consistent name.